### PR TITLE
feat: cart page added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
Cart page is added to this page.
1.  Product card horizontal is added.
2.  cart price details card is added

### CODE REVIEW

File: cart_page/cart_page.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    flex-direction: column;
+    width: 90%;
+    margin: auto;
+    align-items:center;
+}
+
+.cart-container {
+    display: flex;
+    flex-wrap: wrap;
+    /* align-items: center; */
```
Comment (aryan-13) [inline cart_page/cart_page.css:20]: Unwanted comments can be removed

**Review by aryan-13 — APPROVED** [review]
The code looks good

File: cart_page/cart_page.css (hunk 1 above)
Comment (ShreyashMahajan) [reply cart_page/cart_page.css:20]: ok, i will remove them

**Review by cjinc121 — APPROVED** [review]
code is clean as well as ui , looks good

**Pull request merged**